### DISCUSSION TITLE
login does not work telnet ssh SIAB

### DISCUSSION THREAD

**Discussion by georg3003** [Freetz-NG/freetz-ng#867] [Allgemein]
Hello,
on Oct 2nd I downloaded freetz-ng and created for multiple boxes new modified FW. Here I am talking about 7590 and dropbear Telnet and SIAB(Shell in a Box)
After login with root/freetz and exiting login is possible. But after changing the password and exiting and re-login the new login is not possible anymore.

changing the password through Rudi shell is possible, but no logins possible.

Is this a bug? Or How can I delete all the settings and the password for Root.
Entschuldigung mein Englisch ist nicht perfekt.
Grüße  und Danke

**Comment by georg3003** [comment]
Sorry,

forgot

`modusers save
modsave flash`

**Comment by fda77** [comment]
Du kannst auch deutsch schreiben. Probier mal die allerneuest Version von heute, durch tar waren einige Berechtigungen durcheinander, ich weiss nicht seit wann. Dadurch ging zb Dropbear keylogin nicht, nur mit Passwort

**Reply by fda77** [reply]
68eaf28a1502db12002933af6b44c0d14464cc90
